bridgeifns.py

```diff
@@ -0,0 +1,194 @@
+# stdlib
+from os import name
+from typing import Tuple
+# lib
+from cloudcix.rcc import comms_ssh, CouldNotConnectException
+# local
+
+
+__all__ = [
+    'build',
+    'scrub',
+]
+
+SUCCESS_CODE = 0
+
+
+def build(
+    bridgename: str,
+    namespace: str,
```

Comment:
```suggestion
    namespace: str,
    config_file=None
```

An optional path the `config_file` is required in the contract.  Please refer to  the [directorymain](https://github.com/CloudCIX/primitives/blob/87224b988c89702c81770ee3f32d91bf0b7cca3b/directorymain.py#L64) primitive for loading the file and determining the podnet IPs and which one is currently enabled. 

This primitive will run the commands on both the PodNets in HA ([six layer diagram](https://docs.cloudcix.com/understanding/robot_introduction.html))